host_guest/cyclodextrin_derivatives/README.md

```diff
@@ -6,7 +6,7 @@ For SAMPL7, the Gilson group is contributing binding data for two hosts (r-riman
 
 The challenge consists of predicting binding free energies of two guests to the series of modified cyclodextrin hosts shown below.
 
-![](images/sampl_host_structures.jpg)
+![](images/sampl_host_structures_coded.jpg)
```

Comment:
I don't see that you added this file, just changed the link.

Reply:
Also: If you DO add the file, would be good to replace the old file rather than adding a new file (which would just mean there are two different images of the same thing which can get confusing).

Reply:
@davidlmobley Ok I added the the new images and deleted the old. Should be good now